src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,46 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
The `alive` array is declared as a static property of the `Animal` class. Ensure that this is intended, as it will be shared across all instances of `Animal` and its subclasses.